src/gui/editors/PianoRoll.cpp

```diff
@@ -4279,6 +4306,17 @@ void PianoRoll::zoomingChanged()
 }
 
 
+void PianoRoll::zoomingYChanged()
+{
+	s_keyLineHeight = m_zoomYLevels[m_zoomingYModel.value()] * DEFAULT_KEY_LINE_HEIGHT;
+	m_octaveHeight = s_keyLineHeight * KeysPerOctave;
+	m_whiteKeySmallHeight = round(s_keyLineHeight * 1.5);
+	m_whiteKeyBigHeight = round(s_keyLineHeight * 2);
+	m_blackKeyHeight = round(s_keyLineHeight * 1.3333);
```

Comment:
The same for here.

Reply:
See comment above, this is recalculated only once when zoom changes so the overhead is small.